docker/reasonable-adjustments/src/config.json

```diff
@@ -0,0 +1,72 @@
+{
+    "mockDirectory": "./src/mocks",
+    "quiet": false,
+    "port": "9000",
+    "latency": 50,
+    "logRequestHeaders": false,
+    "webServices": {
+        "_status": {
+            "latency": 20,
+            "verbs": ["get"],
+            "responses": {
+                "get": {"httpStatus": 200, "mockFile": "status.json"}
+            }
+      },
+      "consent": {
+          "latency": 20,
+          "verbs": ["get","post"],
+          "headers": {
+            "NHSD-Session-URID": "test",
+            "X-Request-ID": "test"
+          },
+          "responses": {
+                "get": {"httpStatus": 200, "mockFile": "readConsent.json"},
+                "post": {"httpStatus": 201, "mockFile": "createConsent.json"}
+          }
+      },
+      "consent/:consentID": {
+        "latency": 20,
+        "verbs": ["put"],
+        "responses": {
+              "put": {"httpStatus": 200, "mockFile": "updateConsent.json"}
+        }
+    },
+    "flag": {
+        "latency": 20,
+        "verbs": ["get","post"],
+        "responses": {
+              "get": {"httpStatus": 200, "mockFile": "readFlag.json"},
+              "post": {"httpStatus": 201, "mockFile": "createFlag.json"}
+        }
+    },
+    "flag/:flagID": {
+      "latency": 20,
+      "verbs": ["put"],
+      "responses": {
+            "put": {"httpStatus": 200, "mockFile": "updateFlag.json"}
+        }
+  },
+  "list": {
+      "latency": 20,
+      "verbs": ["get","post"],
+      "responses": {
+            "get": {"httpStatus": 200, "mockFile": "readList.json"},
+            "post": {"httpStatus": 201, "mockFile": "createList.json"}
+        }
+  },
+  "list/:listID": {
+    "latency": 20,
+    "verbs": ["put"],
+    "responses": {
+          "put": {"httpStatus": 200, "mockFile": "updateList.json"}
+        }
+    },
+   "removerarecord": {
```

Comment:
Can we rename this to cameCase?

Reply:
This file isn't used; also the endpoint isn't came case